src/test/java/io/jenkins/plugins/logintheme/LoginThemeTest.java

```diff
@@ -0,0 +1,70 @@
+package io.jenkins.plugins.logintheme;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+import org.junit.Test;
+import org.junit.ClassRule;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.junit.Before;
+import java.io.IOException;
+
+import org.kohsuke.stapler.StaplerRequest;
+import net.sf.json.JSONObject;
+
+public class LoginThemeTest {
+
+    @ClassRule
```

Comment:
It's correct but I would prefer JUnit5 tests and using `@WithJenkins` on the class and `org.junit.jupiter.api.Test` annotation

Reply:
Initially, I used JUnit-5 tests, but `ConfigurationAsCodeTest` uses JUnit-4, I thought there would some reason to use it. That's why I opt to JUnit-4.

If there is no specific reason, then I will switch to JUnit-5

Reply:
You are right, but if possible for new tests let's try to use JUnit5 directly to avoid migrate them in the future.

JcasC test hardness actually already support JUnit5 extension : https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/test-harness/src/main/java/io/jenkins/plugins/casc/misc/junit/jupiter/WithJenkinsConfiguredWithCode.java